pyhap/hap_server.py

```diff
@@ -664,13 +664,23 @@ def _handle_remove_pairing(self, tlv_objects):
 
     def handle_resource(self):
         """Get a snapshot from the camera."""
-        if not hasattr(self.accessory_handler.accessory, 'get_snapshot'):
+
+        data_len = int(self.headers["Content-Length"])
+        request_body = self.rfile.read(data_len)
+        data = json.loads(request_body.decode("utf-8"))
+
+        if self.accessory_handler.accessory.category == CATEGORY_BRIDGE:
```

Comment:
Can we check for `STANDALONE_AID` instead?

Reply:
Just tried, but `self.accessory_handler.accessory.aid` returns `1` both for bridge and standalone accessory

Reply:
I've created new pull request which merges correctly with 3.2.0.
https://github.com/ikalchev/HAP-python/pull/306